doc/howto/storage_pools.md

```diff
@@ -13,18 +13,41 @@ See the following sections for instructions on how to create, configure, view an
 LXD creates a storage pool during initialization.
 You can add more storage pools later, using the same driver or different drivers.
 
+`````{tabs}
+````{group-tab} CLI
 To create a storage pool, use the following command:
 
     lxc storage create <pool_name> <driver> [configuration_options...]
+See the {ref}`storage-drivers` documentation for a list of available configuration options for each driver.
 
-Unless specified otherwise, LXD sets up loop-based storage with a sensible default size/quota (20% of the free disk space, but at least 5 GiB and at most 30 GiB).
+````
+
+````{group-tab} UI
+To create a storage pool, select {guilabel}`Pools` from the {guilabel}`Storage` section of the Main navigation.
+
+On the resulting screen, click {guilabel}`Create pool` in the upper right hand corner.
+
+From this screen, you can configure the name and description of your storage pool. A number of other settings can also be configured here depending on the storage driver selected.
+You can also specify which storage driver to use by selecting one from the {guilabel}`Driver` dropdown.
```

Comment:
Maybe switch the last two sentences, so the thought process is a bit more straight forward. Such as first saying "...specify the driver..." and then something like "depending on the driver, you get more configuration options..."